Added design

Jhalakupadhyay opened the pull request:
# Description
Added the Appbar design as you have shown screenshot of if you want any changes in it then please let me know

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)


## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

Roshaen commented:
Hey @Jhalakupadhyay first list your UI component in main ui_collections.dart to render it
and can you please let me know why you have added .jar files?

Jhalakupadhyay commented:
> Hey @Jhalakupadhyay first list your UI component in main ui_collections.dart to render it and can you please let me know why you have added .jar files?

ohhk I will add it to the main.dart and about that .jar file I think because I use to do the backend for android in flutter in java so when I ran the code the android studio has added some file which I don't have any Idea why.

Jhalakupadhyay commented:
Added the design in ui collection and the app worked fine without crashes but while committing it showed conflict in the ui_collection because some other design was there at 18 tag so i changed mine to tag:20 and i hope it will work.
